tests/behat/features/Administrator.feature

```diff
@@ -0,0 +1,170 @@
+@administrator
+Feature: Agency Administrator role
+  In order to keep Agencies, Components, and Managers up to date
+  As an Agency Administrator
+  I should be able to administer Agency Manager user accounts, agencies, and agency components
+
+  @api
+  Scenario: Agency Administrator can administer user accounts with the Agency Manager role
+    Given I am logged in as a user with the 'Agency Administrator' role
+    And I am at 'admin/structure/taxonomy/manage/agency/add'
+    And for 'Name' I enter 'A Test Agency'
+    When I press the 'Save' button
+    Then I should see the following success messages:
+      | Created new term A Test Agency. |
+    When I am at 'admin/people/create'
+    And for 'Username' I enter 'Alex'
+    And for 'Password' I enter 'abc123!@#'
+    And for 'Confirm password' I enter 'abc123!@#'
+    And I check the box 'Agency Manager'
+    And for 'Agency' I enter 'A Test Agency'
+    And I press the 'Create new account' button
+    Then I should see the following success messages:
+      | Created a new user account for Alex. No email has been sent. |
+    When I am at 'admin/people'
+    Then I should see 'Agency Manager' in the 'Alex' row
+    When I click 'Edit' in the 'Alex' row
+    And I press the 'Save' button
+    Then I should see the following success messages:
+      | The changes have been saved. |
+    When I click 'Edit' in the 'Alex' row
+    And I press the 'Cancel account' button
+    And I press the 'Cancel account' button
+    Then I should see the following success messages:
+      | Alex has been disabled. |
+    And the user 'Alex' is deleted
```

Comment:
is this supposed to verify that the user was deleted? or is it supposed to delete the user?

Reply:
It is supposed to delete the user as part of cleanup.